api/src/main/java/au/org/aodn/nrmn/restapi/service/SurveyIngestionService.java

```diff
@@ -216,7 +216,10 @@ public void ingestTransaction(StagedJob job, Collection<StagedRowFormatted> vali
         var messages = Arrays.asList(rowCount + " rows of data", siteCount + " sites", surveyCount + " surveys", obsItemCount + " distinct observable items", diverCount + " divers");
         var message = messages.stream().collect(Collectors.joining("\n"));
 
-        stagedJobLogRepository.save(StagedJobLog.builder().stagedJob(job).details(message).eventType(StagedJobEventType.INGESTED).build());
+        stagedJobLog.setDetails(message);
+        stagedJobLog.setEventType(StagedJobEventType.INGESTED);
```

Comment:
is there a possibility that the ingestion failed and the job log still gets updated as `INGESTED`?